lua/weapons/weapon_Auto_Shotgun.lua

```diff
@@ -0,0 +1,135 @@
+-- Easy Settings part
+
+-- SWEP Info
+SWEP.Author = "yousifgaming"
+SWEP.Instructions = "Primary is the same as Secondary fire automatic shotgun"
+SWEP.Purpose = "To play with Dingus"
+SWEP.Contact = "Don't Contact me i dont like people"
+
+-- SWEP info in spawnmenu
+SWEP.PrintName = "Auto Shotgun"
+SWEP.Category = "yousifgaming's Weapons"
+SWEP.Spawnable = true
+SWEP.AdminOnly = false
+SWEP.IconOverride = "materials/cat.jpg"

```

Comment:
Probably would be better to namespace this material by putting it in a folder named the same as the weapon. Like `materials/autoshotgun/cat.jpg`

Also I think it's better to name it icon.jpg than cat.jog